Resources/js/translator.js

```diff
@@ -53,6 +53,17 @@ var Translator = (function(document, undefined) {
         var _locale = locale || currentLocale || localeFallback,
             _domain = domain;
 
+        /**
+         * If the _locale looks like nl_NL or nl-NL
+         * Try the language as the locale
+         */
+        if (undefined == _messages[_locale]) {
+            var tryLanguage = _locale.match(/^([a-z]{2})(_|-)[A-Z]{2}$/)
```

Comment:
I suggest looking at the logic used in Symfony and porting it to JS. The goal is to have it behave the same than in PHP, so it would make sense